SPECS/git/git.spec

```diff
@@ -1,13 +1,18 @@
 Summary:        Fast distributed version control system
 Name:           git
 Version:        2.23.3
-Release:        2%{?dist}
+Release:        3%{?dist}
 License:        GPLv2
 URL:            https://git-scm.com/
 Group:          System Environment/Programming
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Source0:        https://www.kernel.org/pub/software/scm/git/%{name}-%{version}.tar.xz
+# CVE-2018-1000110, CVE-2018-1000182, CVE-2019-14860, CVE-2020-2136 apply to Jenkins Git Plugin
```

Comment:
These mismatches are caused by tooling issues, as the CVE belongs to the vendor `jenkins` instead of `apache` so we should avoid nopatching them.

Reply:
Vendor to the git entry will be updated in the tooling to avoid nopatching.